### PR TITLE
Create class Car

### PR DESCRIPTION
**Pull request by tiagoxdca**

### CODE REVIEW

File: Car.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    var numberOfDoors: Double?
+    var color: String?
+    var numberOfSeats: String?
+    var price: Double?
+    var brand: String?
+    var model: String?
+    var velocity: Double?
+
+    init(numberOfDoors: Double, color: String, numberOfSeats: String, price:Double, brand: String, model: String, velocity: Double) {
+        self.numberOfDoors = numberOfDoors
```
Comment (tiper) [inline Car.swift:11]: Meh

File: Car.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        self.color = color
+        self.numberOfSeats = numberOfSeats
+        self.price = price
+        self.brand = brand
+        self.model = model
+        self.velocity = velocity
+    }
+
+    func incrementVelocity() {
+        velocity += 5
```
Comment (tiper) [inline Car.swift:21]: Meh meh

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+*.java
```
Comment (tiper) [inline .gitignore]: Please consider also adding .class and .kt

File: Car.swift (hunk 1 above)
Comment (tiagoxdca) [reply Car.swift:11]: Bah

File: Car.swift (hunk 2 above)
Comment (tiagoxdca) [reply Car.swift:21]: Bah bah

File: .gitignore (hunk 3 above)
Comment (tiagoxdca) [reply .gitignore]: :+1:

**Review by tiper — APPROVED** [review]

**Pull request merged**